docs/pages/reference/cli.mdx

```diff
@@ -67,6 +71,29 @@ clusters.
 | `--app-uri` | none | none | Internal address of the application to proxy |
 | `--app-public-addr` | none | none | Public address fo the application to proxy |
 
+#### teleport start --roles
+
+The `--roles` flag when used with `teleport --start` instructs Teleport on which specific Teleport services to start. Below is a more cohesive table of roles and their associated services that `teleport start` supports:
+
+| Service | Role Name | Description |
+| - | - | - |
+| [Node](../architecture/nodes.mdx) | `node` | Runs a daemon on a host that allows SSH connections from authenticated clients. |
+| [Auth](../architecture/authentication.mdx) | `auth` | Authenticates hosts and users who want access to Teleport-managed resources or information about a cluster. |
+| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
+| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
```

Comment:
```suggestion
| [App](../application-access/introduction.mdx) | `app` | Provides access to applications. |
```

Remove "daemon", and I'm not sure the reverse tunnel is relevant info - kind of an implementation detail, although if you want to emphasize the consequence of that detail (Teleport only requires the proxy service to be externally reachable thanks to reverse tunnels) then that would be useful info.